### DISCUSSION TITLE
Generate Label dialog empty when labels/reports are in app resources

### DISCUSSION THREAD

**Discussion by bronwyncombs** [specify/specify7#5839] [Feature Requests]
**Describe the bug**
Generate label button on form inaccurately shows "No Results" for reports and labels.

**To Reproduce**
1. Go to https://herbrbge33123-v79-dev.test.specifysystems.org/specify/
2. Find a CO form
3. Scroll down to `Generate Label`

**Expected behavior**

**Screenshots**
![Screen Shot 2023-07-06 at 3 55 18 PM](https://github.com/specify/specify7/assets/135047322/4f8966d2-06e9-458a-b0ce-4b7629254494)
![Screen Shot 2023-07-06 at 3 59 56 PM](https://github.com/specify/specify7/assets/135047322/26481288-9075-4433-a585-5195facf8d36)
![Screen Shot 2023-07-06 at 3 55 47 PM](https://github.com/specify/specify7/assets/135047322/4bd70a7f-ac60-4f90-96ec-4ba4513ddc5b)

**Desktop:**
 - OS: macOS
 - Browser: [e.g. Chrome
 - Specify 7 Version: 7.9-dev

**Database Name:** If applicable
herb_rbge

**Reported By**
Name of your institution

**Additional context**

Can produce in older versions on this database in particular. Issue doesn't present in other databases in 7.9-dev or earlier versions.

**Comment by maxpatiiuk** [comment]
If you click the `Generate label` dialog on the form, then only labels for current table (in your case Collection Object) are displayed

If you open "Reports" tool, labels for all tables are displayed

We could clarify this by replacing "No results" with "No results for Collection Object table"
